### DISCUSSION TITLE
Help with dropdowns - manual options

### DISCUSSION THREAD

**Discussion by MrWombastic** [Budibase/budibase#5880] [Help]
Hi, 
In reference to https://github.com/Budibase/budibase/discussions/2483#discussioncomment-2732792

I am having questions. In the related post your first dropdown isn't based in a table, there are entered Manual options.

I am having the same schema as you regarding the 2 tables, but the difference is, that the column I want to filter the second dropdown isnt type text, it's relationship. For example:
Table SubAssets:
image
Table Assets:
image

So first option picker:
I pick "Silber"
Second option picker should only show values associated with the Asset "Silber" (1st screenshot).

Filtering the data provider for the SubAsset option picker (2nd) seems to be a problem, because column with type relationship is not visible:
image

Am I wrong?

And if type relationship is not possible: I saw that your first dropdown isn't based on the diagnostics table, instead of choosing the values from this table, you make an option field with fixed values to choose of. That's ok for a short formula, but if you got 50 options in our table, you wont update the dropdown manually every time.
Think of a table where a user could enter new data to fill the dropdown, I don't see how to realize it with this solution.
Am I wrong?

**Comment by mike12345567** [comment]
Hi @MrWombastic - I've converted your issue to a discussion for visibility by the support team. Issues should follow our bug or feature request templates.

**Comment by melohagan** [comment]
Hi @MrWombastic 

Filtering on Relationships isn't currently possible: https://github.com/Budibase/budibase/issues/5585
Filtering is also not available for multi-select: https://github.com/Budibase/budibase/issues/3773

There is a bit of a work around here: https://github.com/Budibase/budibase/discussions/5493#discussioncomment-2631134

It's a bit convoluted, but basically if <a href="https://docs.budibase.com/docs/actions#on-change">On Change</a> of the dropdown, you save the relationship data to a <a href="https://docs.budibase.com/docs/json">JSON Array</a>, then you can use that JSON Array column as a data source to display only the chosen relationship data.
